components/brave_rewards/resources/tip/components/one_time_tip_form.tsx

```diff
@@ -36,13 +28,12 @@ function generateTipOptions (
 
 function getDefaultTipAmount (
   rewardsParameters: RewardsParameters | undefined,
-  publisherInfo: PublisherInfo | undefined,
   balanceInfo: BalanceInfo | undefined
 ) {
-  if (!rewardsParameters || !publisherInfo || !balanceInfo) {
+  if (!rewardsParameters || !balanceInfo) {
     return 0
   }
-  const options = generateTipOptions(rewardsParameters, publisherInfo)
+  const options = generateTipOptions(rewardsParameters)
```

Comment:
We should probably remove the `amounts` property from the `PublisherInfo` interface if we are no longer using it. We should also remove that property from the data structure created in `brave_tip_ui.cc` as well.

Reply:
Removed, thanks!